lib/block/header.go

```diff
@@ -2,32 +2,31 @@ package block
 
 import (
 	"encoding/json"
-	"time"
 
 	"boscoin.io/sebak/lib/voting"
 )
 
 type Header struct {
 	// TODO rename `Header` to `BlockHeader`
-	Version          uint32    `json:"version"`
-	PrevBlockHash    string    `json:"prev_block_hash"`   // TODO Uint256 type
-	TransactionsRoot string    `json:"transactions_root"` // Merkle root of Txs // TODO Uint256 type
-	Timestamp        time.Time `json:"timestamp"`
-	Height           uint64    `json:"height"`
-	TotalTxs         uint64    `json:"total-txs"`
-	TotalOps         uint64    `json:"total-ops"`
+	Version          uint32 `json:"version"`
+	PrevBlockHash    string `json:"prev_block_hash"`   // TODO Uint256 type
+	TransactionsRoot string `json:"transactions_root"` // Merkle root of Txs // TODO Uint256 type
+	ProposedTime     string `json:"proposed-time"`
```

Comment:
👍

Reply:
I would like to use `_` in place of `-` in json.  like `transactions_root`  :-)

Reply:
It was my mistake ^^;; I prefer that too.

There are many of this case. use `-` T^T
For this case, I changed `-` to `_`
I will make another PR to refactor remains. @anarcher